### DISCUSSION TITLE
How to create own NeRF dataset?

### DISCUSSION THREAD

**Discussion by SsNiPeR1** [NVlabs/instant-ngp#476] [Q&A] [answered]
I cannot find a way how to generate transforms.json. Is there a way to do it, or I have to write it myself?

**Comment by sepigraph** [comment, marked as answer]
No. Read the documentation.

[https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md](url)

**Reply by RobTranquillo** [reply]
you put the wrong url into your link. `https://github.com/NVlabs/instant-ngp/discussions/url` must be 
[https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md](https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md)
